Project-Euler/Project-Euler/DynamicProblemPrinter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_Euler
+{
+    class DynamicProblemPrinter
+    {
+        public static void PrintProblem(int problemNumber, Type objectType)
+        {
+            var method = objectType.GetMethod($"Problem{problemNumber}");
+
+            method.Invoke(null, null);
```

Comment:
It might also be nice to wrap this in a timer so the user knows how long each method took. i.e.

Record current time
Invoke Method
Print current time - start time

Reply:
I like that idea. However, I will add this later on once I have added more problems.